### PR TITLE
editoast: handle empty object case in auto fixes

### LINKED ISSUE #5867: Magic wand: Empty object
# Description

One sub-task of the magic wand issue #5521.

In case of `Empty object` error, the magic wand is supposed to remove the object.

The work done in https://github.com/osrd-project/osrd/pull/5673 should be used as guideline.

# Goal

Save much time when fixing errors in infra.

# Acceptance criteria

- [ ] In case of `Empty object` error, the magic wand is supposed to remove the object.

### PR DESCRIPTION
**Pull request by hamz2a**
closes #5867

### CODE REVIEW

File: editoast/src/views/infra/auto_fixes.rs [hunk 1]
```diff
@@ -503,4 +509,71 @@ mod test {
...
+        );
+
+        let operational_point = OperationalPoint {
+            parts: vec![],
+            ..Default::default()
+        };
+        let operational_point_railjson = RailjsonObject::OperationalPoint {
+            railjson: operational_point.clone(),
+        };
+        let req_create = get_create_operation_request(operational_point_railjson, small_infra_id);
```
Comment (flomonster) [inline editoast/src/views/infra/auto_fixes.rs]:
```suggestion
        let operational_point: RailjsonObject = OperationalPoint::default().into();
        let req_create = get_create_operation_request(operational_point, small_infra_id);
```

**Review by woshilapin — APPROVED** [review]

File: editoast/src/views/infra/auto_fixes.rs [hunk 2]
```diff
@@ -503,4 +509,71 @@ mod test {
...
+        );
+
+        let speed_section = SpeedSection {
+            track_ranges: vec![],
+            ..Default::default()
+        };
+        let speed_section_railjson = RailjsonObject::SpeedSection {
+            railjson: speed_section.clone(),
+        };
+        let req_create = get_create_operation_request(speed_section_railjson, small_infra_id);
```
Comment (flomonster) [inline editoast/src/views/infra/auto_fixes.rs]:
```suggestion
        let speed_section: RailjsonObject = SpeedSection::default().into();
        let req_create = get_create_operation_request(speed_section, small_infra_id);
```

File: editoast/src/views/infra/auto_fixes.rs [hunk 3]
```diff
@@ -503,4 +509,71 @@ mod test {
...
+        let small_infra_id = small_infra.id();
+
+        let catenary = Catenary {
+            track_ranges: vec![],
+            ..Default::default()
+        };
+        let catenary_railjson = RailjsonObject::Catenary {
+            railjson: catenary.clone(),
+        };
+        let req_create = get_create_operation_request(catenary_railjson, small_infra_id);
```
Comment (flomonster) [inline editoast/src/views/infra/auto_fixes.rs]:
```suggestion
        let catenary: RailjsonObject = Catenary::default().into();
        let req_create = get_create_operation_request(catenary, small_infra_id);
```

**Review by flomonster — APPROVED** [review]
LGTM. Some code can be factorized

**Review by bougue-pe — APPROVED** [review]

**Pull request merged**